package.json

```diff
@@ -26,6 +26,7 @@
   },
   "homepage": "https://github.com/ubclaunchpad/foodies#readme",
   "dependencies": {
+    "@googlemaps/google-maps-services-js": "3.1.16",
```

Comment:
This is a great library https://googlemaps.github.io/google-maps-services-js/index.html. It's basically a wrapper around axios, but the good thing about it is **types**. We don't need to create our own interfaces.